### DISCUSSION TITLE
Why is BT-271 not rendered on the preview?

### DISCUSSION THREAD

**Discussion by Wout3P** [OP-TED/eForms-SDK#362] [Q&A] [answered]
As stated in the title.  When generating a preview through the render api.
Why is BT-27 rendered but BT-271 not?

**Comment by rouschr** [comment]
Hello, 

please always provide concerned SDK version and notice sub type.
The SDK version and the notice sub type can be found inside the concerned notice.

You can find the SDK here: https://github.com/OP-TED/eForms-SDK, for the versions look into tags next to the branches.
If you look into the SDK `/view-templates` folder, there are .efx files, in some of them you will see text like:
`... {BT-27-Procedure} #{name|BT-27}: ...`
1.efx, 2.efx, ... correspond to the related notice sub type 1, 2, ...
So if you do not find BT-271 in the concerned efx file it will not be displayed.
Now maybe BT-271 should be displayed according to you, I am not a business person who can decide about this, but we could consider adding it if it makes sense.

**Comment by Wout3P** [comment, marked as answer]
Hello thanks for helping, 
It's concerning the last version of the sdk 1.5 and notice subtype 16.
Indeed BT-271 is not mentioned in these .efx files.
Since it is possible to send this information in the xml according to the fields.json I would asume it should be rendered?
According to a business point of view I think it can make sence to render this information?

**Reply by rouschr** [reply]
Someone selected what is displayed.
Maybe @grecoca or @pdonohoe can decide if BT-271 should/could be displayed for notice sub type 16.

**Reply by ferrakaop** [reply]
BT-271 Framework Maximum Value is one of the new fields introduced by the 2022 amendment. We are working on adding these fields to the view-templates and it should be available in SDK 1.7.
